<launch>

	<arg name="init_speed" default="3.0" />
	<arg name="decelate_speed" default="2.0" />
	<arg name="accelate_speed" default="5.0" /> 
	<arg name="backward_movement_speed" default="-2.0" /> 
	<arg name="init_lookahead_distance" default="7.0" />
	<arg name="decelate_lookahead_distance" default="2.5" />
	<arg name="accelate_lookahead_distance" default="10.0" />
	<arg name="current_mission_state" default="0" />
	<arg name="first_state_index" default="5" />
	<arg name="second_state_index" default="9" />
	<arg name="third_state_index" default="17" />
	<arg name="fourth_state_index" default="28" />
	<arg name="fifth_state_index" default="1130" />
	<arg name="sixth_state_index" default="1140" />
	<arg name="seventh_state_index" default="1150" />
	<arg name="eighth_state_index" default="1160" />
	<arg name="nineth_state_index" default="1170" />
	<arg name="lane_final" default="0" />

	<node name="waypoint_follower_node" pkg="waypoint_follower" type="waypoint_follower" output="screen">
		<param name="init_speed" value="$(arg init_speed)" />
		<param name="decelate_speed" value="$(arg decelate_speed)" />
		<param name="accelate_speed" value="$(arg accelate_speed)" />
		<param name="backward_movement_speed" value="$(arg backward_movement_speed)" />
		<param name="init_lookahead_distance" value="$(arg init_lookahead_distance)" />
		<param name="decelate_lookahead_distance" value="$(arg decelate_lookahead_distance)" />
		<param name="accelate_lookahead_distance" value="$(arg accelate_lookahead_distance)" />
		<param name="current_mission_state" value="$(arg current_mission_state)" />
		<param name="first_state_index" value="$(arg first_state_index)" />
		<param name="second_state_index" value="$(arg second_state_index)" />
		<param name="third_state_index" value="$(arg third_state_index)" />
		<param name="fourth_state_index" value="$(arg fourth_state_index)" />
		<param name="fifth_state_index" value="$(arg fifth_state_index)" />
		<param name="sixth_state_index" value="$(arg sixth_state_index)" />
		<param name="seventh_state_index" value="$(arg seventh_state_index)" />
		<param name="eighth_state_index" value="$(arg eighth_state_index)" />
		<param name="nineth_state_index" value="$(arg nineth_state_index)" />
		<param name="lane_final" value="$(arg lane_final)" />
	</node>

</launch>s
